.github/workflows/Bit.Client.Web.BlazorUI.Action.yml

```diff
@@ -0,0 +1,25 @@
+name: BlazorUI
+on:
+  push:
+    paths:
+    - 'src/Client/Web/**'
+jobs:
+  build:
+    name: Test successful build and Run Tests
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 5.0.x
+    - name: Restore dependencies
+    #I handled execution with "find | grep | xarg", to allow later bulk operations, with reducing the regex limits
```

Comment:
If simple execution is preferred, inform me to update. I wrote it for cases when all tests and all projects in the folder require build/pass check. Then added BlazorUI in the pattern to let it be specific for now.

Reply:
I think ```dotnet test``` step is enough because it will restore nuget packages and build project dependencies
It's also fine to hard-code project name and path

Reply:
You are correct. There is still a reason to justify the separation of commands. When a PR fails the test, it's faster to monitor how this PR is failed (in which step). Still, if this isn't worth the separation, inform me to update the file.